website/content/v1.8/talos-guides/install/bare-metal-platforms/equinix-metal.md

```diff
@@ -93,11 +93,12 @@ This guide assumes the user has a working API token,and the [Equinix Metal CLI](
 ```bash
 metal device create \
   --project-id $PROJECT_ID \
-  --facility $FACILITY \
+  --metro $METRO \
   --operating-system "custom_ipxe" \
   --ipxe-script-url "https://pxe.factory.talos.dev/pxe/376567988ad370138ad8b2698212367b8edcb69b5fd68c80be1f2ec7d603b4ba/{{< release >}}/equinixMetal-amd64" \
-  --plan $PLAN\
-  --hostname $HOSTNAME\
+  --always-pxe \
```

Comment:
once installed to disk doesn't need to pxe again

Reply:
I agree with @frezbo, this is definitely wrong, it would prevent Talos upgrades from working properly